appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/model/media/ContentImpl.java

```diff
@@ -60,6 +62,8 @@ public class ContentImpl extends LinkedHashMap<String, MediaType> implements Con
 
     private static final long serialVersionUID = 1575356277308242221L;
 
+    private Map<String, Object> extensions = new HashMap<>(); // workaround as Content doesn't extend Extendable!
```

Comment:
Is this fine to not be null? You drew attention to the extension map needing to be null previously in another class

Reply:
Actually it is ok, because the whole stuff around "null" is just to solve TCK requirement, which is discussed and will be probably removed (usage of `@Extension` on method without being part of `@APIResponse(s)`).
Anyway, I set to to null, there may be some other case I don't see now.